WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -253,7 +291,7 @@ import WordPressShared
         featuredImageView.image = nil
         var url = featuredImageURL
         featuredImageDesiredWidth = featuredImageView.frame.width
-        let size = CGSize(width:featuredImageDesiredWidth, height:featuredMediaHeightConstraintConstant)
+        let size = CGSize(width:featuredImageDesiredWidth, height:100)
```

Comment:
Maybe a `const` for `100`, or maybe it should read the `constant` value for the featured media height constraint?